src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -143,6 +144,21 @@ void OnSystemRequestNotification::AddHeader(BinaryMessage& message) const {
   LOG4CXX_DEBUG(
       logger_, "Header added: " << std::string(message.begin(), message.end()));
 }
+
+std::string OnSystemRequestNotification::ParsePTString(std::string& pt_string) const{
+  std::string result;
```

Comment:
@JackLivio In fact I believe you can work with the same string instead of copy. The string class has the `insert` method which is I believe fit you purpose